src/main/java/com/hms/flexyosisoftconnector/dataserver/OSIsoftServer.java

```diff
@@ -25,15 +25,15 @@
 
   private static boolean connected = true;
 
-  public static final int LINK_ERROR = 32601;
+  static final int LINK_ERROR = 32601;

```

Comment:
commit says add public but this is removing it, was this intentional?

Reply:
no, this line should not be touched here. it gets removed in a later commit, changing this

Reply:
`public` is still being removed here in the 'Added missing javadocs to OSIsoftServer' commit